GVRf/Framework/framework/src/main/java/org/gearvrf/scene_objects/GVRTextViewSceneObject.java

```diff
@@ -93,6 +112,11 @@
     private int mCount;
     private final GVRDrawFrameListenerImpl mFrameListener;
 
+    // individual lines of text.  Assists in determining lenghts of
+    // strings once the fonts are set.
+    private String linesOfText[] = null;
+    private GVRMaterial gvrMaterialX3D = null;
```

Comment:
Why do we need a special material for just one case? It doesn't seem necessary to me.